run

```diff
@@ -47,32 +40,54 @@ fi
 
 # Parse arguments
 UPDATE_ENV=false
+LOCAL=false
 ARGS=()
 
 for arg in "$@"; do
     case $arg in
-        --update-env)
+        update-env)
```

Comment:
I believe we could also run `./run local --update-env` that will still work, right?

Reply:
So I removed this because it was doing the same thing as `./run update-env` and it didn't feel like we needed two ways to update the env. If this is a dealbreaker let me know and I can try and find a way to add it back. I am just sort of on a quest so that if you want to update-env there is one and only one way to do that.

Reply:
Not a problem for me, but some on the team may say something because this may be functionality they're used to using.

Reply:
Sounds good. I'll manage that fall out :)